### PR TITLE
Allow setting trusted certificate variable.

### PR DESCRIPTION
**Pull request by anuraaga**
I was originally hoping this would solve https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1829 but realized that's about mTLS, not TLS. I don't think the spec defines variables for mTLS yet (technically the current variable is misdocced I guess). But now that I made this change it doesn't hurt to get it in and the variable will likely be renamed in the future.

### CODE REVIEW

File: sdk-extensions/autoconfigure/build.gradle [hunk 1]
```diff
@@ -81,6 +85,12 @@ testJaeger {
     environment("OTEL_BSP_SCHEDULE_DELAY_MILLIS", "10")
 }
 
+testOtlpTls {
+    environment("OTEL_RESOURCE_ATTRIBUTES", "service.name=test,cat=meow")
+    environment("OTEL_TRACE_EXPORTER", "otlp")
+    environment("OTEL_BSP_SCHEDULE_DELAY_MILLIS", "10")
```
Comment (jkwatson) [inline sdk-extensions/autoconfigure/build.gradle:91]: this shouldn't have MILLIS any more, correct?

Comment (jkwatson) [reply sdk-extensions/autoconfigure/build.gradle:91]: also the ones above and below.

Comment (jkwatson) [reply sdk-extensions/autoconfigure/build.gradle:91]: and yes, I missed this in the original PR to remove the MILLIS.  😢

File: sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java [hunk 2]
```diff
@@ -59,6 +63,21 @@ static OtlpGrpcSpanExporter configureOtlpSpans(ConfigProperties config) {
       builder.setTimeout(Duration.ofMillis(timeoutMillis));
     }
 
+    String certificate = config.getString("otel.exporter.otlp.certificate");
```
Comment (jkwatson) [inline sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java:66]: the spec also suggests span and metric specific configuration. do we need to support that, or is this good enough (I think this is good enough for now...we can add the signal-specific ones later if needed)

File: sdk-extensions/autoconfigure/build.gradle (hunk 1 above)
Comment (jkwatson) [reply sdk-extensions/autoconfigure/build.gradle:91]: https://github.com/open-telemetry/opentelemetry-java/pull/2574

**Review by jkwatson — APPROVED** [review]

File: sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java (hunk 2 above)
Comment (anuraaga) [reply sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java:66]: Yup missing the signal-specific ones for all these variables, will add in followup

**Pull request merged**